pd/src/verify.rs

```diff
@@ -29,8 +29,12 @@ pub struct VerifiedTransaction {
     pub new_notes: BTreeMap<note::Commitment, NoteData>,
     /// List of spent nullifiers from spends in this transaction.
     pub spent_nullifiers: BTreeSet<Nullifier>,
-    /// Net delegations performed in this transaction.
+    /// Net delegations performed in this transaction. An identity key mapped to zero is different
+    /// from an identity key that is absent; the former indicates that a validator's net change in
+    /// delegation in this transaction was zero *but it experienced some (un)delegations*.
```

Comment:
style nit: could "An identity key..." move down a paragraph, so it's easier to parse the "summary / details" format while scanning the code?

(good observation though)

Reply:
Thanks, fixed.